docs/user/security/rbac_tutorial.asciidoc

```diff
@@ -0,0 +1,119 @@
+
+* Create a role named `mortgage-developer`
+* Give the role the ability to monitor the Elastic Stack
+* Give the role permission to access the data in the relevant indices
+* Give the role permission to create visualizations and dashboards 
+* Create the web developer's user account with the proper roles
+
+[float]
+==== Create a role named `mortgage-developer`
+
+Go to **Management > Roles** 
+for an overview of your roles.  This view provides actions
+for you to create, edit, and delete roles.
+
+[role="screenshot"]
+image::security/images/role-management.png["Role management"]
+
+
+You can create as many roles as you like. Click *Create role* and 
+provide a name; 
+in this tutorial use `dev-mortgage` because this role is for a 
+developer working on the bank's mortgage application.
+
+[float]
+==== Give the role the ability to monitor the Elastic Stack
+
+Stack monitoring is an important feature for system admins and developers.  
+Both of these groups of people will use stack monitoring to look at 
+performance.  Developers will also use stack monitoring to evaluate and tune ingest pipelines.  For more information on stack monitoring see the https://www.elastic.co/guide/en/elasticsearch/reference/current/monitoring-overview.html[monitoring overview].
+
+Stack monitoring is a cluster level privilege, so in *Cluster 
+privileges* select  **monitor**
+
+[role="screenshot"]
+image::security/images/role-monitor-privilege.png["Monitor privilege"]
+
+[float]
+==== Give the role permission to access the data
+
+Access to data in indices is an index level privilege, so in 
+*Index privileges* add a line or lines for the indices containing the 
+data for this role.  There are two privileges needed, `read` and 
+`view_index_metadata`.  All of the privileges are detailed in the 
+https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html[security privileges] documentaion.
+In the screenshots Filebeat and Metricbeat data is used, but you 
+should use the index patterns for your indices.
+
+[role="screenshot"]
+image::security/images/role-index-privilege.png["Index privilege"]
+
+[float]
+==== Give the role permission to create visualizations and dashboards
+
+By default, roles do not give Kibana privileges.  Click **Add space 
+privilege** and associate this role with the `Dev Mortgage` space
+
+[role="screenshot"]
+image::security/images/role-space-associate.png["Associate space"]
+
+To create new visualizations and dashboards users with 
+the `dev-mortgage` role need read and write (All) access to 
+**Visualize** and **Dashboard**.  **Discover** has also 
+been given read and write access because it is common for developers 
+to create saved searches while designing visualizations.
+
+[role="screenshot"]
+image::security/images/role-space-visualization.png["Associate space"]
+
+[float]
+==== Create the developer's user account with the proper roles
+
+Create a user with the `dev-mortgage` role and the `monitoring-user` 
```

Comment:
Should we tell the user where to the UI is for creating a role.

Reply:
Good catch!